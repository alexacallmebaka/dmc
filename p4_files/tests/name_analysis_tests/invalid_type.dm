glob : void;
b : string;
f : (arg1 : void,arg2 : void) int {
}