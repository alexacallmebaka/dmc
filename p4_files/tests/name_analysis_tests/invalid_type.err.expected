FATAL [1,1]-[1,5]: Invalid type in declaration
FATAL [2,5]-[2,11]: Invalid type in declaration
FATAL [2,1]-[2,2]: Invalid type in declaration
FATAL [3,6]-[3,10]: Invalid type in declaration
FATAL [3,18]-[3,22]: Invalid type in declaration
Name Analysis Failed